services/QuillLMS/.puppeteerrc.cjs

```diff
@@ -0,0 +1,8 @@
+const {join} = require('path');
+
+/**
+* @type {import("puppeteer").Configuration}
+* */
```

Comment:
Is this commented out?  Should we just remove it?

Reply:
This was a snippet from [step 4](https://github.com/Studiosity/grover#running-on-heroku) of "Running on Heroku". Given the formatting of the comment, it might be used to autogenerate stuff elsewhere so I'd be inclined to leave it.